spec/features/journeys_spec.rb

```diff
@@ -45,6 +45,21 @@ def sign_in(user)
     expect(page.current_url).to eq(faqs_url)
   end
 
+  scenario 'As an anonymous user I want to check my journeys in process' do
+    visit user_journeys_path
```

Comment:
Je to v menu vobec viditelne, ked clovek nie je prihlaseny? Ja by som to do menu nedaval.

Reply:
Viditeľné to samozrjme nie je, ale ak by sa náhodou chcel cez url k tomu dostať aj neprihlásený, tak som testovala ako sa aplikácia zachová v takom prípade.

Reply:
Aha, ok potom ale trosku inak nazvime ten test, lebo nie je z neho zrejme co je ocakavany vysledok. Kedze toto je vlastne fail a nie happy path, ktore sa pri feature testoch nezvyknu robit.